File: workshop4.js
class Student {
    static school_name = "WE-School";  

    constructor(name, grade) {
        this.name = name;  
        this.grade = grade;  
    }

    display() {
        console.log(`Name: ${this.name}, Grade: ${this.grade}, School: ${Student.school_name}`);
    }
}


let student1 = new Student("Ahmed", "2nd");
let student2 = new Student("Ali", "3rd");


student1.display();
student2.display();

File: workshop4.py
class student:
    school_name="weschool"
    def __init__(self,name,grade):
        self.name=name
        self.grade=grade 
    def display(self):
        print(f"Name: {self.name}, Grade: {self.grade}, School: {self.school_name}")

student1 = student("Ahmed", "2nd")
student2 = student("Ali", "3rd")

student1.display()
student2.display()

